cfme/utils/tests/test_soft_assert.py

```diff
@@ -89,3 +89,13 @@ def test_soft_assert_helpers(soft_assert):
 
     # the caught_asserts identifier is now empty after calling clear_asserts
     assert not caught_asserts
+
+
+@pytest.fixture(scope="function")
+def some_fixture(soft_assert):
+    soft_assert(False, "bla bla bla")
+
+
+def test_soft_assert_fail_in_fixture(some_fixture):
```

Comment:
shouldnt this one be marked as xfail in some way if its supposed to be a "broken" test